EidSuggest/Suggest.php

```diff
@@ -25,43 +25,37 @@
 
 # TSFE initialization
 
-tslib_eidtools::connectDB();
-$pageId     = filter_var(t3lib_div::_GET('id'), FILTER_SANITIZE_NUMBER_INT);
+$pageId     = filter_var(\TYPO3\CMS\Core\Utility\GeneralUtility::_GET('id'), FILTER_SANITIZE_NUMBER_INT);
 $languageId = filter_var(
-	t3lib_div::_GET('L'),
+	\TYPO3\CMS\Core\Utility\GeneralUtility::_GET('L'),
```

Comment:
True you can import without class too, it is just not generally done in the core.
But I guess that's mostly because most of the times it is required in ext_* or TCA and those are merged during caching.
But in the dumpFile eID it isn't done with imports either but that could just be because of conversion from old classnames to new ones.